Sources/Data Conversion.swift

```diff
@@ -53,6 +53,7 @@ extension BigUInt {
         let byteCount = (self.bitWidth + 7) / 8
 
         let buffer = UnsafeMutableBufferPointer<UInt8>.allocate(capacity: byteCount)
+        buffer.initialize(repeating: 0)
```

Comment:
Works, but setting the remaining bytes (`0..<i`) to zero after the loop below could be more efficient.

Reply:
good thinking :)